crypto/conf/conf_lib.c

```diff
@@ -356,10 +356,10 @@ OPENSSL_INIT_SETTINGS *OPENSSL_INIT_new(void)
 {
     OPENSSL_INIT_SETTINGS *ret = malloc(sizeof(*ret));
 
-    if (ret != NULL)
+    if (ret != NULL) {
```

Comment:
I would write it as:
```
if (ret == NULL)
    return NULL
```

It should probably at least use OPENSSL_malloc(), and you can even replace it by OPENSSL_zalloc() and remove the memset call below.

Reply:
At first, I think the same.
Them will editing it, I read the comment block just above ;)